packages/http-client-java/generator/http-client-generator-test/src/main/java/com/_specs_/azure/core/traits/models/RepeatabilityResult.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com._specs_.azure.core.traits.models;
+
+/**
+ * Repeatability Result header options.
+ */
+public enum RepeatabilityResult {
```

Comment:
Caused by this case: https://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#1-responses-header-schema-has-public-access-in-tcgc